lib/macho/utils.rb

```diff
@@ -11,6 +11,13 @@ def self.round(value, round)
       value
     end
 
+    # @param size [Fixnum] the pre-padded size
```

Comment:
I think “unpadded size” should be clearer; “pre-padded” sounds to me like there's already some padding in place.